README.rst

```diff
@@ -12,7 +12,14 @@ Directory Structure
     │   ├── main.py
     │   ├── pyproject.toml
     │   └── poetry.lock
-    │
+    ├── infra
+    │   ├── app.py
```

Comment:
That might be too verbose. 
Consider adding description to each folder instead: backend, infra, tests. That would bring some clarity and benefits for new users

Reply:
fixed it